test/rack/prx_auth/token_data_test.rb

```diff
@@ -48,5 +48,59 @@
       assert !token.authorized?(789)
     end
 
+    describe 'with wildcard role' do
+      let(:aur) { {'*' => 'peek', '123' => 'admin', '456' => 'member' } }
+
+      it 'applies wildcard tokens to queries with no matching aur' do
+        assert token.authorized?(789, :peek)
+      end
+
+      it 'does not authorize unscoped for wildcard resources' do
+        assert !token.authorized?(789)
+      end
+
+      it 'allows querying by wildcard resource directly' do
+        assert token.authorized?('*', :peek)
+        assert !token.authorized?('*', :admin)
+      end
+
+      it 'has a shorthand `gobally_authorized?` to query wildcard' do
+        assert token.globally_authorized?(:peek)
+        assert !token.globally_authorized?(:admin)
+      end
+
+      it 'treats global authorizations as additive to other explicit ones' do
+        assert token.authorized?(123, :peek)
+      end
+
+      it 'refuses to run `globally_authorized?` with no scope' do
+        assert_raises ArgumentError do
+          token.globally_authorized?
```

Comment:
I'm curious why you want to disallow a scopeless globally authorized check.  Just to keep things explicit?

Reply:
Oh, i see.  I should have read the description.  Nevermind!